tests/unit/cli/test_main.py

```diff
@@ -26,6 +27,20 @@ def test_cli_some_command(self):
             result = runner.invoke(cli, ["local", "generate-event", "s3"])
             self.assertEqual(result.exit_code, 0)
 
+    def test_cli_with_no_region_arg_validate(self):
+        mock_cfg = Mock()
+        with patch("samcli.cli.main.GlobalConfig", mock_cfg):
+            runner = CliRunner()
+            runner.invoke(cli, ["validate", "--region", "--debug"])
+            self.assertRaises(RegionError)
+
+    def test_cli_with_no_region_arg_deploy(self):
+        mock_cfg = Mock()
+        with patch("samcli.cli.main.GlobalConfig", mock_cfg):
+            runner = CliRunner()
+            runner.invoke(cli, ["deploy", "use-container" "False", "--region", "--debug"])
```

Comment:
It seems like both tests are using an empty region since another flag is provided right after `--region`? I would rather have;
- A test with a valid region
- A test with an invalid region
- A test with empty region